configuration/concepts/adult_hiv_initial_form.csv

```diff
@@ -0,0 +1,97 @@
+db3088d9-cc5c-433f-8e47-5e29881dd9c5,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e;Do Not Know,Cotrimoxazole (Prophylaxie en cours),Cotrimoxazole (Prophylaxsis in progress),,,Misc,Coded,,
+2fba5a24-fd7c-4f15-808e-ffafdeebb786,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,Allergie médicamenteuse,Medication allergy,,,Misc,Coded,,
+d0b941d1-cc7d-4fe1-9a7c-188a858431d1,,,Si oui précisez (exemple: Allergie au cotrimoxazole),If yes specify allergy,,,Misc,Text,,
+cf74d45d-7a99-4c12-9bbb-02095726f666,fd43f382-0f14-4740-ad8f-3ec38b05fd14;02f7b1ff-1bfc-46b8-811d-bf524db681cb;fef74daa-2a47-4c4e-b9c2-d18078d8e3be;81ff1578-da33-44f3-bdbe-71d7706ecd87;aeecf2e8-4746-4023-bc39-efdd83ce4e94;3b2a2ded-60f6-4818-a4f4-10f296f53ae1;29538d59-7239-475d-b95c-3e493f8b88fe;db3088d9-cc5c-433f-8e47-5e29881dd9c5;2fba5a24-fd7c-4f15-808e-ffafdeebb786;d0b941d1-cc7d-4fe1-9a7c-188a858431d1,,III.3 ARV ET Autres,III.3 ARV AND OTHER,,,Misc,N/A,,
+d25b1be5-157a-4d2c-b29b-f872b231416d,,,GPAA (Gravité Parté Accouchement),GPAA (Severity released childbirth),,,Misc,Text,,
+0336010a-4aae-4112-b1a7-5a29d1724517,,,DDR,DDR,,,Misc,Text,,
+279583bf-70d4-40b5-82e9-6cb29fbe00b4,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,Grossesse,Pregnancy,,,Misc,Coded,,
+83f3b54f-fa55-4bea-b73f-7e3aa8b6a1e2,,,Ménopause,Menopause,,,Misc,N/A,,
+fb9b3d62-fb72-4b62-95c8-b3f3e5eaf92e,,,Trouble régles,Difficulties,,,Misc,N/A,,
+c45f6876-3f10-11e4-adec-0800271c1b75,,,Normale,Normal,,,Misc,N/A,,
+8e4bdf44-636c-4af2-81e8-16b22b37c89e,,83f3b54f-fa55-4bea-b73f-7e3aa8b6a1e2;fb9b3d62-fb72-4b62-95c8-b3f3e5eaf92e;Normal,Menstruation,Menstruation,,,Misc,Coded,,
+4c2629fe-5b75-4721-aec2-d5b792c7eb8e,d25b1be5-157a-4d2c-b29b-f872b231416d;0336010a-4aae-4112-b1a7-5a29d1724517;Pregnancy;8e4bdf44-636c-4af2-81e8-16b22b37c89e,,III.4 Chez la femme,III.4 Female,,,Misc,N/A,,
+7d1187b3-f428-4064-80ec-869c16761793,23d94813-c123-4f6d-a7f8-66e22bc30e75;9a7ad1b1-942f-43b4-b280-5df81b515135;cf74d45d-7a99-4c12-9bbb-02095726f666;4c2629fe-5b75-4721-aec2-d5b792c7eb8e,,III. ANTECEDENTS DU PATIENT,III. PATIENT BACKGROUND,,,Misc,N/A,,
+e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,,,Date de début de traitement ARV,ARV treatment start date,,,Misc,Date,,
+20b2eac4-ff3a-4569-b610-067c4cd8b027,e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,,Traitements,Treatments,,,Misc,N/A,,
+059b8309-4cba-46cc-a857-e1e74be3fa86,20b2eac4-ff3a-4569-b610-067c4cd8b027,,IV. VISITE INITIALE,IV. INITIAL VISIT,,,Misc,N/A,,
+1fb2dd86-53b5-4815-9c64-edc081b908d9,83e4df0e-5723-4c53-ae8b-7411b2337bef;7d1187b3-f428-4064-80ec-869c16761793;059b8309-4cba-46cc-a857-e1e74be3fa86,,FICHE DU PATIENT AVEC VIH - ADULTE,PATIENT WITH HIV - ADULT INITIAL,,,Misc,N/A,,
+9d91cb77-3f10-11e4-adec-0800271c1b75,18351dd1-078e-456c-8c23-915b16e9dfaf;c36a7537-3f10-11e4-adec-0800271c1b75;1fb2dd86-53b5-4815-9c64-edc081b908d9,,All Observation Templates,All Observation Templates[FR],,,ConvSet,N/A,,

```

Comment:
@lauravignoli this change will be overridden in `/child_hiv_initial_form.csv`, I would like to suggest we remove it here